nnpackage/examples/v1.3.0/two_circles/metadata/MANIFEST

```diff
@@ -0,0 +1,11 @@
+{
+  "major-version" : "1",
+  "minor-version" : "3",
+  "patch-version" : "0",
+  "configs"       : [  ],
+  "models"        : [ "mv1.0.circle", "mv1.1.circle" ],
```

Comment:
Note: It has only circle models. It does not have trix model. See https://github.com/Samsung/ONE/issues/9206#issuecomment-1148276287 for information about these two models.